mappings/net/minecraft/command/CommandBase.mapping

```diff
@@ -1,27 +1,162 @@
 CLASS aq net/minecraft/command/CommandBase
+	CLASS aq$1
+		METHOD a get (Lave;)Z
+			ARG 0 blockState
+	CLASS aq$2
+		METHOD a get (Lave;)Z
+			ARG 0 blockState
+	CLASS aq$a NormalizedBlockPosSelector
+		FIELD a originalPos D
+		FIELD b selectorPos D
+		FIELD c middle Z
+		METHOD <init> (DDZ)V
+			ARG 0 originalPos
+			ARG 1 selectorPos
+			ARG 2 middle
+		METHOD a getOriginalPos ()D
+		METHOD b getSelectorPos ()D
+		METHOD c getMiddle ()Z
 	FIELD a commandManager Lap;
 	FIELD b COMMA_SPLITTER Lcom/google/common/base/Splitter;
 	FIELD c KEY_VALUE_SPLITTER Lcom/google/common/base/Splitter;
 	METHOD a getMinPermissionLevel ()I
+	METHOD a normalizeBlockPosSelector (DLjava/lang/String;IIZ)Laq$a;
+		ARG 0 originalPos
+		ARG 1 selector
+		ARG 2 min
+		ARG 3 max
+		ARG 4 middle
+	METHOD a normalizeBlockPosSelector (DLjava/lang/String;Z)Laq$a;
+		ARG 0 originalPos
+		ARG 1 selector
+		ARG 2 middle
 	METHOD a toBlockState (Lank;Ljava/lang/String;)Lave;
+		ARG 0 block
+		ARG 1 meta
 	METHOD a setCommandManager (Lap;)V
+		ARG 0 commandManager
 	METHOD a compareTo (Las;)I
+		ARG 0 command
+	METHOD a getPlayerFromSender (Lau;)Lnn;
+		ARG 0 sender
 	METHOD a dispatch (Lau;Las;ILjava/lang/String;[Ljava/lang/Object;)V
 	METHOD a dispatch (Lau;Las;Ljava/lang/String;[Ljava/lang/Object;)V
+	METHOD a giveItem (Lau;Ljava/lang/String;)Lahd;
+		ARG 0 sender
+		ARG 1 itemName
+	METHOD a toBlockPos (Lau;[Ljava/lang/String;IZ)Ldx;
+		ARG 0 sender
+		ARG 1 params
+		ARG 2 startIndex
+		ARG 3 middle
+	METHOD a getStateWithProperty (Lave;Lavu;Ljava/lang/Comparable;)Lave;
+		ARG 0 blockState
+		ARG 1 propertyKey
+		ARG 2 propertyValue
+	METHOD a toBlockPropertyvalue (Lavu;Ljava/lang/String;)Ljava/lang/Comparable;
+		ARG 0 blockProperty
+	METHOD a getCommandJsonException (Lcom/google/gson/JsonParseException;)Ldq;
+		ARG 0 parseException
 	METHOD a toInteger (Ljava/lang/String;)I
+		ARG 0 toConvert
 	METHOD a toDoubleWithMin (Ljava/lang/String;D)D
+		ARG 0 toConvert
+		ARG 1 min
 	METHOD a toDoubleWithBounds (Ljava/lang/String;DD)D
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a toIntegerWithMin (Ljava/lang/String;I)I
+		ARG 0 toConvert
+		ARG 1 min
 	METHOD a toIntegerWithBounds (Ljava/lang/String;II)I
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a toLongWithBounds (Ljava/lang/String;JJ)J
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a matchesBeginning (Ljava/lang/String;Ljava/lang/String;)Z
+		ARG 0 string
+		ARG 1 toFind
+	METHOD a joinPrettyStringCollection (Ljava/util/Collection;)Ljava/lang/String;
```

Comment:
This name is a big long, as well as the other methods with similar names - any reason this isn't just `joinPretty`? The signature doesn't match.